common/src/main/java/io/homeassistant/companion/android/database/wear/Caches.kt

```diff
@@ -0,0 +1,19 @@
+package io.homeassistant.companion.android.database.wear
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+/**
+ * Represents a favorite entity
+ */
+@Entity(tableName = "cache")
+data class Caches(
```

Comment:
The idea was to start out with just caching favorites, as I currently don't see the need to cache any other entities. However, a general name like `cache` provides the future possibility of caching other entities as well. Let me know what you think.

The comment on line 7-9 is mere a silly mistake by me not paying attention when copying the contents of the `Favorites.kt` file, sorry for that haha, fixing as we speak :)

Reply:
> However, a general name like cache provides the future possibility of caching other entities as well. 

Correct, but this data class is specifically for favorited entities. If we want to cache something else in the future, other data/properties will have to be stored so probably a new data class would be used instead of adding it to this existing data class.